consensus/istanbul/validator/default.go

```diff
@@ -279,7 +279,7 @@ func (valSet *defaultSet) CalcProposer(lastProposer common.Address, round uint64
 	defer valSet.validatorMu.RUnlock()
 
 	if len(valSet.validators) == 0 {
-		logger.Error("len of validators is 0, Proposer is nil", "validators", valSet.validators)
+		logger.Error("List of validators is empty", "validators", len(valSet.validators))
```

Comment:
```suggestion
		logger.Error("List of validators is empty, Proposer is nil")
```
I guess that this error print like "List of validators is empty validators 0". What about my suggestion?

Reply:
Just logging `Validiator list is empty` is not bad, but I thought showing the actual value would not be bad as well. As the validator list is zero, the suggested `Proposer is nil` makes sense. However, the logging of nil proposer is already located in other places. Either one is okay for me. The code owner may add a further suggestion.

Reply:
I agree your opinion, too. How about add just a comma in "List of validators is empty validators 0" like "List of validators is empty , validators 0"?

Reply:
If the log does not contain the concrete value printing, simply `Validator list is empty` seems an appropriate candidate I think.

Reply:
It's good to me, also.